wger.py

```diff
@@ -127,186 +161,188 @@ def create_exercise(self):
         category_id = str([i['id'] for i in categories if i['name'] == category][0])
         equipment_id = str([i['id'] for i in equipments if i['name'] == equipment][0])
         muscle_id =str([i['id'] for i in muscles if i['name'] == muscle][0])
-        data={'name':name, 'description':description, 'category':category_id, 'equipment': equipment_id, 'muscles':muscle_id,
+        data={'name':name,
+              'description':description,
+              'category':category_id,
+              'equipment': equipment_id,
+              'muscles':muscle_id,
               'license_author': author}
         response=self.post_info('exercise/',data)
         return response
 
-    def select_exercises(self,workout,day):
+    def select_exercises(self,workout,day,total_ex,ex_musclegroup):
```

Comment:
Don't be afraid to use longer variable names, as long as they serve a better role of being specific and explicit(no issue in using total_exercise_number and muscle_group_exercise_numer).